src/utils/authHelpers.jsx

```diff
@@ -0,0 +1,31 @@
+import firebase from "../firebaseConfig";
+
+export const EMAIL_NOT_FOUND_ERROR = "auth.emailNotFound";
+export const EMAIL_ALREADY_TAKEN_ERROR = "auth.emialAlreadyTaken";
+export const PASSWORD_INCORRECT_ERROR = "auth.incorrectPassword";
+
+export function setFirebaseProvider(providerName) {
+  let providerSet;
+  switch (providerName) {
+    case "google":
+      providerSet = new firebase.auth.GoogleAuthProvider();
+      break;
+    case "facebook":
+      providerSet = new firebase.auth.FacebookAuthProvider();
+      break;
+    case "twitter":
+      providerSet = new firebase.auth.TwitterAuthProvider();
+      break;
+    default:
+      throw new Error("provider not specified!");
+  }
+
+  return providerSet;
+}
+
+export default {
+  setFirebaseProvider,
+  EMAIL_NOT_FOUND_ERROR,
```

Comment:
you already exported them

Reply:
I used these as a default so I can do something like:
```
import from "../../utils/authHelpers";

authHelpers.setFirebaseProvider
```

What do you think? keep it or remove it? @louisrli